### PR TITLE
Created files for VodQA application testing and implemented scroll() method

### PR DESCRIPTION
**Pull request by mukundgpt2025**
Added feature, steps, BL, screens (abstract and android implementation) for VodQA application. Added code for scroll(fromPoint, toPoint) in Driver class according to latest Appium version.

### CODE REVIEW

File: src/test/resources/com/znsio/teswiz/features/vodqa.feature [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@vodqa
+
+Feature: Vodqa test
+
+  @android
+  Scenario: Scrolling using fromPoint and toPoint
```
Comment (raghavgarg1996) [inline src/test/resources/com/znsio/teswiz/features/vodqa.feature:6]: Scrolling using - looks very odd.

it should be like - Validating scroll functionality in the App

File: src/main/java/com/znsio/teswiz/runner/Driver.java [hunk 2]
```diff
@@ -130,14 +133,16 @@ public WebElement findElementByXpath(String locator) {
...
-//                   .waitAction(WaitOptions.waitOptions(Duration.ofMillis(1000)))
-//                   .moveTo(PointOption.point(toPoint))
-//                   .release()
-//                   .perform();
+        AppiumDriver appiumDriver = (AppiumDriver) this.driver;
+        PointerInput touch = new PointerInput(PointerInput.Kind.TOUCH, "touch");
+        Sequence scroller = new Sequence(touch, 1);
+        scroller.addAction(touch.createPointerMove(Duration.ofSeconds(0), PointerInput.Origin.viewport(), toPoint.getX(), toPoint.getY()));
+        scroller.addAction(touch.createPointerDown(PointerInput.MouseButton.LEFT.asArg()));
+        scroller.addAction(touch.createPointerMove(Duration.ofMillis(700), PointerInput.Origin.viewport(), fromPoint.getX(), fromPoint.getY()));
```
Comment (anandbagmar) [inline src/main/java/com/znsio/teswiz/runner/Driver.java:141]: What is this 700 magic number?

File: src/test/java/com/znsio/teswiz/steps/VodQASteps.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    @Given("I login to vodqa application using credentials")
+    public void loginToApplication() {
+        Drivers.createDriverFor(SAMPLE_TEST_CONTEXT.ME, Runner.getPlatform(), context);
+        new VodqaBL(SAMPLE_TEST_CONTEXT.ME, Runner.getPlatform()).login();
+    }
+
+    @When("I click on Vertical Swiping")
+    public void selectVerticalSwipingTile() {
```
[the remark this answers is not in the document]
Comment (mukundgpt2025) [reply src/test/java/com/znsio/teswiz/steps/VodQASteps.java]: This is already removed

[the remark this answers is not in the document]
Comment (anandbagmar) [reply src/test/java/com/znsio/teswiz/steps/VodQASteps.java]: I still see it here

**Pull request merged**